tests/test_browser.py

```diff
@@ -4204,7 +4204,7 @@ def test_small_js_flags(self):
     size = os.path.getsize('test.js')
     print('size:', size)
     # Note that this size includes test harness additions (for reporting the result, etc.).
-    self.assertLess(abs(size - 5368), 100)
+    self.assertLess(abs(size - 5508), 100)
```

Comment:
How much of this change is due to this PR itself? I'd hope none, but if so then it could be separate so the squashed commit doesn't make the PR look guilty?

(Or is it related?)

Reply:
This is because of the increase in the size of browser_reporting.js.. not the size of the actual runtime.   I'll create an seperate PR to update to the exact current value though before I land this.

Reply:
Yup, rebaselining first means this is needed at all.